src/lib/core/CHIPConfig.h

```diff
@@ -2498,6 +2498,15 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_VERBOSE_VERIFY_OR_DIE 0
 #endif
 
+/**
+ * @def CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_DEVICES
+ *
+ * @brief Number of devices a controller can be simultaneously connected to
+ */
+#ifndef CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_DEVICES
+#define CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_DEVICES 64
```

Comment:
Did you have to push this down in your examples? What's the value you use?

Reply:
We have a very basic light switch sample in our SDK for which it's enough to allocate space for one device. With 64 devices the controller wouldn't fit in 256KB RAM by quite a big margin.